common/browser.go

```diff
@@ -391,6 +394,15 @@ func (b *Browser) newPageInContext(id cdp.BrowserContextID) (*Page, error) {
 
 // Close shuts down the browser.
 func (b *Browser) Close() {
+	if b.closed {
+		b.logger.Warnf(
+			"Browser:Close",
+			"Please call browser.close only once, and do not use continue using the browser after calling close.",
```

Comment:
There is a typo in this message right?
I guess it should be:
```
Please call browser.close only once, and do not use the browser after calling close.
```
Maybe even omitting the last part of `calling close` or changing it for `after it`? I'm not native English speaker, so just suggesting :smile:

Reply:
@ka3de Rephrased like a native speaker here: https://github.com/grafana/xk6-browser/pull/803/commits/250b1e79838c9d636b5e8061ef35b180b1809ca9 😅